package.json

```diff
@@ -22,10 +22,14 @@
     ]
   },
   "scripts": {
-    "start": "npm run prepare && npm run style && npm run web",
-    "build": "npm run prepare && npm run style && browserify app.js | indexhtmlify --title 'peermaps' > public/index.html",
-    "prepare": "mkdir -p public && cp node_modules/eyros/2d.wasm public/eyros2d.wasm",
-    "web": "budo app.js -l -d public",
-    "style": "georender-style2png node_modules/georender-style2png/example/style.json -o public/style.png"
+    "start": "npm run prepare && npm run web",
+    "build": "npm run prepare && npm run browserify",
+    "prepare": "npm run prepare:config && npm run prepare:public && npm run prepare:style && npm run prepare:wasm",
+    "prepare:config": "cp -n config.default.json config.json",
```

Comment:
`cp -n` for not overwriting `config.json` if it already exists (so others can override)